dask/dataframe/io/tests/test_json.py

```diff
@@ -120,6 +121,26 @@ def _my_json_reader(*args, **kwargs):
         assert_eq(actual, actual_pd)
 
 
+@pytest.mark.parametrize(
+    "engine",
+    [
+        pytest.param(
+            "ujson",
+            marks=pytest.mark.skipif(
+                parse_version(pd.__version__) < parse_version("2.0"),
```

Comment:
We've got a compatibility variable for this

https://github.com/dask/dask/blob/0890b96be75e46ef2b88c11015761b9a6bd32480/dask/dataframe/_compat.py#L19